cogs/clash_river_war.py

```diff
@@ -139,6 +139,30 @@ def remaining_war_embed(all_clan_attacks):
     embed.add_field(name=columns, value=row_val, inline=False)
     return embed
 
+def players_war_attacks(all_current_war_players: PlayersRemainingWarAttacks):
+    embed = discord.Embed(
+        description=dedent('''
+            **Current River Race Players War Attacks **
+            **decks Used Today** - How many decks the player used today (maximum of 4)
+            **In Clan** - Checking whether is currently in the clan
+            **Player Name** - Accumulated Medals throughout the River Race

```

Comment:
Typo? The field is "Player Name", but the description here is talking about medals?

Reply:
Also, you called it "Player Name" here but "Name" down in the below output. Please change to make both consistent with each other

Reply:
done